sources/assets/Xenko.Core.Assets.CompilerApp/build/Xenko.Core.Assets.CompilerApp.targets

```diff
@@ -135,4 +135,14 @@
       </None>
     </ItemGroup>
   </Target>
+
+  <!--Feels like we could do this here, but it does not appear to work-->
```

Comment:
Please remove that from here (it's a batch file, never going to work)

Reply:
Left this for discussion only to see your thoughts.